msg/Position.msg

```diff
@@ -0,0 +1,3 @@
+float64 latitude

```

Comment:
We should use [geographic_msgs/GeoPoint](http://docs.ros.org/en/noetic/api/geographic_msgs/html/msg/GeoPoint.html) instead of recreating